packages/material-ui-lab/src/SliderUnstyled/ValueLabel.js

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import clsx from 'clsx';
+import { withStyles } from '@material-ui/core/styles';
```

Comment:
`SliderUnstyled` imports this component. Shouldn't `ValueLabel` also be "unstyled"? 

That's probably why `SliderUnstyled` is so big. It's as big as the old `Slider`. `SliderStyled` is even bigger than `Slider`.

Reply:
"requested changes" for visibility. This is good to go IMO. Don't have an overview anymore anyway :wink:

Reply:
Good catch! Will get on it tomorrow 👍

Reply:
Implemented with - https://github.com/mui-org/material-ui/pull/22435/commits/bb9fc680d79b00432aa08d7bf73738b23ebbfc1c